plugin.xml

```diff
@@ -86,7 +86,7 @@
 
         <framework src="push.gradle" custom="true" type="gradleReference" />
         <framework src="com.android.support:support-v13:23+" />
-        <framework src="com.google.android.gms:play-services-gcm:+" />
+        <framework src="com.google.android.gms:play-services-gcm:9.0.2" />
```

Comment:
Can you make this 9.0.2+ so it will accept future versions of play services instead of being pinned to 9.0.2?